components/Header/index.js

```diff
@@ -1,7 +1,22 @@
+import Link from "next/link";
+import styled from "styled-components";
+
 export default function Header() {
   return (
     <header>
       <h1>Endstagram</h1>
+      <Button>
+        <Link href="/">Home</Link>
+      </Button>
+      <Button>
+        <Link href="/preferences-form">Preferences</Link>
+      </Button>
```

Comment:
```suggestion

        <Link href="/">Home</Link>
  
     
        <Link href="/preferences-form">Preferences</Link>

```
It's not considered good practice to wrap a Button around a link, as they are both interactive elements that have different purposes. You can however styles you links with the same styling as you would give a button